### DISCUSSION TITLE
Small Achievements Makes A Difference

### DISCUSSION THREAD

**Discussion by verma-kunal** [EddieHubCommunity/support#1504] [Show and tell]
Contributed to @nhcarrigan's Discord Bot project.
Even though it was a little help I could do, I don't know why but this gave me more confidence towards contributing to open-source.
Small achievements count !!

Shared on Twitter
Link: https://twitter.com/kverma_twt/status/1389664690305867777?s=20

**Comment by eddiejaoude** [comment]
Awesome 👍

**Comment by naomi-lgbt** [comment]
Definitely not a small achievement - your first contribution to the project, *and* you solved a bug that's been plaguing me for a month. 🙂

**Reply by verma-kunal** [reply]
Thank you !! Looking forward to help again !!😃
